ec/acc/deployment_basic_defaults_test.go

```diff
@@ -37,11 +37,9 @@ func TestAccDeployment_basic_defaults(t *testing.T) {
 	startCfg := "testdata/deployment_basic_defaults_1.tf"
 	secondCfg := "testdata/deployment_basic_defaults_2.tf"
 	thirdCfg := "testdata/deployment_basic_defaults_3.tf"
```

Comment:
Looks like we're still using `testdata/deployment_basic_defaults_3.tf` as the `startCfg` in line `:139`, is that intended? If so LGTM

Reply:
Yes, it's intended as all the other configurations have kibana and/or apm instances which aren't really necessary for this test. I'm avoiding starting up instances we don't need. :)